app/src/main/res/drawable/ic_qrcode_scan.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:height="24dp"
+    android:tint="#FFFFFF"
```

Comment:
This also changes the tint of the icon in other places of the app, like the floating action button.